synonymdev/bitkit-react-native
# fix(transfer): update liquidity policy

### Description

- update LSP channel balances according to [new policy](https://docs.google.com/document/d/199MW_Zxm8xm0XMBhSE3feSxikYv1oW0D_OHMCrk94q4/edit?tab=t.0#heading=h.yyauh5flml4m)
- show split network and service fees for CJIT

### Linked Issues/Tasks

Closes https://github.com/synonymdev/bitkit/issues/1856 https://github.com/synonymdev/bitkit/issues/2054 https://github.com/synonymdev/bitkit/issues/2379

### Type of change


### Tests


### QA Notes

- although we decided to remove the minimum for the LSP balance there is still a technical minimum required for LDK of 2% of total channel size, so it cannot be 0
- there's also required minimum channel size from blocktank (currently 20k sats on production), in some cases more is added on the LSP side to fulfill this requirement
- we still subtract a buffer of 2% of the maximum channel size from blocktank to account for exchange rate fluctuation
- there is still a 80% cap on the max client balance to account for onchain fees
- sometimes you have to add/subtract the reserve balance (1%) for channel balances to get the numbers you expect
- currently there is an issue where CJIT "you will receive" amount is wrong
